carma_cloud_client/include/carma_cloud_client/carma_cloud_client_config.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+/*
+ * Copyright (C) 2022 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <iostream>
+#include <vector>
+
+namespace carma_cloud_client
+{
+
+  /**
+   * \brief Stuct containing the algorithm configuration values for carma_cloud_client
+   */
+  struct Config
+  {
+    
+    std::string url ="http://127.0.0.1:"; 
```

Comment:
yaml provides the desired value at runtime. The config struct is used combine all the configurable parameters in one place, instead of defining them one by one in header file